.gitattributes

```diff
@@ -9,3 +9,18 @@ Assets/Music/510953__theojt__cinematic-battle-song.mp3 filter=lfs diff=lfs merge
 Assets/Music/510953__theojt__cinematic-battle-song.mp3.meta filter=lfs diff=lfs merge=lfs -text
 Assets/Music/A1-Cave.mp3 filter=lfs diff=lfs merge=lfs -text
 Assets/Music/A1-Cave.mp3.meta filter=lfs diff=lfs merge=lfs -text
```

Comment:
is this still necessary?

Reply:
Yeah mayb, as I said below, never wrote in the file manually.